cmd/vic-machine/create/create.go

```diff
@@ -239,6 +241,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Virtual Container Host x509 certificate file",
 			Destination: &c.cert,
 		},
+		cli.StringFlag{
+			Name:        "base-image-size",
```

Comment:
Honestly it's confusing to me too but I have been struggling to think of a better name -- paging @hickeng